### PR TITLE
WIP: Activate vgs

### PR DESCRIPTION
**Pull request by joseivanlopez**
PBI: https://trello.com/c/ZH73v4E1/528-3-storageng-activate-and-probe-inactive-lvm

This is a WIP. Please, opinions about whether this is the correct form to activate vgs.

### CODE REVIEW

**Comment by aschnell** [comment]
As discussed on IRC this approch will not work in combination with activating LUKS.

File: storage/StorageImpl.cc [hunk 1]
```diff
@@ -62,18 +62,24 @@ namespace storage
 
 	Devicegraph* probed = create_devicegraph("probed");
 
+    // FIXME
+    // This approach for vgs activation does not work in combination with activating
+    // LUKS. Managing lvm with LUKS, raid, etc is pending to implement.
 	switch (environment.get_probe_mode())
 	{
 	    case ProbeMode::STANDARD: {
+	    LvmVg::Impl::activate();
```
Comment (shundhammer) [inline storage/StorageImpl.cc]: Wrong indentation - please fix

Please notice that a tab is 8 blanks (!) according to our coding style. Everything else just breaks everybody's editor.

File: storage/StorageImpl.cc [hunk 2]
```diff
@@ -62,18 +62,24 @@ namespace storage
...
 		probe(probed);
 	    } break;
 
 	    case ProbeMode::STANDARD_WRITE_DEVICEGRAPH: {
+	    LvmVg::Impl::activate();
```
Comment (shundhammer) [inline storage/StorageImpl.cc]: Wrong indentation

File: storage/StorageImpl.cc [hunk 3]
```diff
@@ -62,18 +62,24 @@ namespace storage
...
 		probe(probed);
 		probed->save(environment.get_devicegraph_filename());
 	    } break;
 
 	    case ProbeMode::STANDARD_WRITE_MOCKUP: {
+	    LvmVg::Impl::activate();
```
Comment (shundhammer) [inline storage/StorageImpl.cc]: Wrong indentation

File: storage/Devices/LvmVgImpl.cc [hunk 4]
```diff
@@ -338,6 +338,19 @@ namespace storage
     }
 
 
+    void
+    LvmVg::Impl::activate()
+    {
+    string cmd_line = VGCHANGEBIN " -a y";
```
Comment (shundhammer) [inline storage/Devices/LvmVgImpl.cc]: Wrong indentation

File: storage/Devices/LvmVg.cc [hunk 5]
```diff
@@ -66,6 +66,13 @@ namespace storage
     }
 
 
+    void
+    LvmVg::activate()
+    {
+    return LvmVg::Impl::activate();
```
Comment (shundhammer) [inline storage/Devices/LvmVg.cc]: Wrong indentation

**Comment by aschnell** [comment]
Apart from the fact that the code will need reimplementation soon it technical does fulfill the PBI. So LGTM - A hurrah to SCRUM!

**Comment by shundhammer** [comment]
LGTM

**Pull request merged**